toolchain/gcc-9.5.0-09-sparc-pic_crtstuff.patch

```diff
@@ -1,17 +1,18 @@
-diff -Naur gcc-9.3.0-orig/libgcc/config/sparc/t-phoenix gcc-9.3.0/libgcc/config/sparc/t-phoenix
---- gcc-9.3.0-orig/libgcc/config/sparc/t-phoenix
-+++ gcc-9.3.0/libgcc/config/sparc/t-phoenix
-@@ -0,0 +1,4 @@
+diff -Naur gcc-9.3.0-orig/gcc/config/sparc/t-phoenix gcc-9.3.0/gcc/config/sparc/t-phoenix
```

Comment:
uch, I think this file should remain in `libgcc/config/sparc`

Reply:
yes, it should, thanks